fuse-subgraph/subgraph.template.yaml

```diff
@@ -138,49 +138,6 @@ dataSources:
           handler: handleTokenRegistered
       file: ./src/mappings/entities.ts
 templates:
```

Comment:
you can also delete `templates` I think

Reply:
https://github.com/fuseio/fuse-graph/blob/d129edebf50073334e6c31f3bad0052ebce1b04d/fuse-subgraph/src/mappings/entities.ts#L26 we have this line which depends on this

Reply:
got ya